packages/rn-tester/RNTester/AppDelegate.mm

```diff
@@ -189,9 +189,9 @@ - (NSDictionary *)prepareInitialProps
 - (NSURL *)sourceURLForBridge:(__unused RCTBridge *)bridge
 {
 #if !TARGET_OS_OSX // [macOS]
-  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"packages/rn-tester/js/RNTesterApp.ios"];
+  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"js/RNTesterApp.ios"];
 #else // [macOS
-  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"packages/rn-tester/js/RNTesterApp.macos"];
+  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"js/RNTesterApp.macos"];
```

Comment:
Is this change (and others) also a diff or are they upstream too?

Reply:
This change was made upstream later: https://github.com/facebook/react-native/pull/36686